config/production.france.js

```diff
@@ -5,6 +5,10 @@ const HOST = process.env.HOST || "api.openfisca.fr",
   apiBaseUrl = process.env.API_URL || `https://${HOST}`,
   gitHubProject = "openfisca/openfisca-france",
   gitWebpageUrl = "https://github.com/openfisca/legislation-explorer",
+  piwikConfig = {
+    url: "stats.data.gouv.fr",
+    siteId: 4
```

Comment:
I would love to have [`trackErrors`](https://github.com/joernroeder/piwik-react-router#trackerrors-false) too!